apis/python/src/tiledbsc/uns_array.py

```diff
@@ -89,13 +90,15 @@ def from_numpy_ndarray(self, arr: np.ndarray) -> None:
             # Note arr.astype('str') does not lead to a successfuly tiledb.from_numpy.
             arr = np.array(arr, dtype="O")
 
-        # overwrite = False
-        # if self.exists:
-        #     overwrite = True
-        #     log_io(None, f"{self._indent}Re-using existing array {self.uri}")
-        # tiledb.from_numpy(uri=self.uri, array=arr, ctx=self._ctx, overwrite=overwrite)
-        # TODO: find the right syntax for update-in-place (tiledb.from_pandas uses `mode`)
-        tiledb.from_numpy(uri=self.uri, array=arr, ctx=self._ctx)
+        if self.exists():
+            tiledbsc.logging.logger.info(
+                f"{self._indent}Re-using existing array {self.uri}"
```

Comment:
Maybe _updating_ existing array to be [consistent with R](https://github.com/TileDB-Inc/tiledbsc/blob/6c3331611a4f06558c3c802dcf8813ff0b33bdae/R/AssayMatrix.R#L58-L62)?

Reply:
This is better wording with or without the R precedent!

It was a copypasta from somewhere else, somewhen else -- thanks for the catch 😁